clang/lib/CodeGen/CGClass.cpp

```diff
@@ -2779,13 +2779,47 @@ void CodeGenFunction::EmitTypeMetadataCodeForVCall(const CXXRecordDecl *RD,
   }
 }
 
+std::pair<SanitizerKind::SanitizerOrdinal, llvm::SanitizerStatKind>
+CodeGenFunction::ParseCFITypeCheckKind(CFITypeCheckKind TCK) {
+  SanitizerKind::SanitizerOrdinal M;
+  llvm::SanitizerStatKind SSK;
+
+  switch (TCK) {
+  case CFITCK_VCall:
+    M = SanitizerKind::SO_CFIVCall;
```

Comment:
Why not just `return std::make_pair(SanitizerKind::SO_CFIVCall, llvm::SanStat_CFI_VCall)`?  It's a little more concise and also inline with LLVM's coding style of returning as early as possible.

Reply:
Thanks for the review! I refactored this function into https://github.com/llvm/llvm-project/pull/140117. I'll make the change there.